astronomer/providers/amazon/aws/triggers/s3.py

```diff
@@ -61,6 +63,7 @@ def serialize(self) -> tuple[str, dict[str, Any]]:
                 "hook_params": self.hook_params,
                 "poke_interval": self.poke_interval,
                 "soft_fail": self.soft_fail,
+                "should_check": self.should_check,
```

Comment:
What is `should_check`?

Reply:
just a boolean based on this I'll see if need to return running status from trigger 
see astronomer/providers/amazon/aws/sensors/s3.py L81